package.json

```diff
@@ -17,6 +17,9 @@
     "postmerge": "yarn install",
     "precommit": "git status --porcelain | awk 'match($1, \"[ACM]\"){print $2}' | grep '[js|jsx]$' | xargs eslint",
     "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook-static": "build-storybook -c .storybook -o .out",
```

Comment:
@mattkime `yarn check` only ensures that `yarn.lock` lines up with `package.json` - if a required node module isn't installed at all (e.g. if it was installed globally on a dev machine and not added as a local dependency), then `yarn check` would pass, but this feature would fail when merging to master.